bin/fin

```diff
@@ -2,6 +2,11 @@
 
 FIN_VERSION=1.108.0
 
+# Allow to override Docksal version during install.
+# Install: curl -L https://get.docksal.io | DOCKSAL_VERSION=develop bash
+# Update: DOCKSAL_VERSION=develop fin update
+DOCKSAL_VERSION="${DOCKSAL_VERSION:-v1.16.2}"
```

Comment:
This is the current version. Does it need to be set for the version at the next release? 1.17.0?

Reply:
Yes. This will need to be manually updated right before a release.